.github/workflows/rust.yml

```diff
@@ -0,0 +1,48 @@
+name: rust
+
+concurrency:
+  group: build-${{ github.head_ref }}
```

Comment:
The group should be the same as workflow name. Use `rust-${{ github.head_ref }}` instead.